apps/proxy-auth/src/app/otp/component/login/login.component.html

```diff
@@ -1,9 +1,24 @@
+<div class="dialog-header">
+    <button *ngIf="step !== 1" mat-icon-button class="back-button" (click)="changeStep(1)">
+        <svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 -960 960 960" width="24px" fill="#5f6368">
```

Comment:
Create a separate file for svg icon and use them into img tag <img src="" alt="" loading="lazy"/>